worker/src/RTC/Codecs/VP9.cpp

```diff
@@ -369,12 +372,15 @@ namespace RTC
 						tmpTemporalLayer = context->GetTargetTemporalLayer();
 					}
 				}
+			}
 
-				// Filter temporal layers higher than current one.
-				if (packetTemporalLayer > tmpTemporalLayer)
-				{
-					return false;
-				}
+			// Filter temporal layers higher than current one.
+			uint16_t temporalLayerForPictureId =
+			  isOldPacket ? context->GetTemporalLayerForPictureId(this->payloadDescriptor->pictureId)
+			              : tmpTemporalLayer;
+			if (packetTemporalLayer > temporalLayerForPictureId)
```

Comment:
Please let's add a blank line after variable declarations and before and after condition blocks.
```suggestion
			// Filter temporal layers higher than current one.
			uint16_t temporalLayerForPictureId =
			  isOldPacket ? context->GetTemporalLayerForPictureId(this->payloadDescriptor->pictureId)
			              : tmpTemporalLayer;

			if (packetTemporalLayer > temporalLayerForPictureId)
```